data/json/items/tools.json

```diff
@@ -10797,5 +10797,30 @@
     "ammo": "NULL",
     "revert_to": "null",
     "use_action": "LADDER"
+  },
+  {
+    "id": "iceaxe",
+    "type": "TOOL",
+    "symbol": ";",
+    "color": "dark_gray",
+    "name": "ice axe",
+    "description": "This is an ice axe with hammer on its head, a multi-purpose hiking and climbing tool used by mountaineers.  It is sturdy enough, so you can use it to pry open closed doors or lift manhole covers.",
+    "price": 5000,
+    "material": "steel",
+    "weight": 500,
+    "volume": 3,
+    "bashing": 14,
+    "cutting": 3,
+    "flags" : ["STAB", "DURABLE_MELEE"],
```

Comment:
Well, ```chop``` flag isn't appropriate for this tool, ```stab``` flag gives too small chance of sticking. If I decide not to put any of this flags, then tool will get "cutting" damage, right? This isn't appropriate too.
I think ```stab``` is a the lesser of three evils.